Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -292,8 +293,12 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
 #endif
 
 #if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
-    if (properties.changedProperties & RemoteLayerTreeTransaction::EventRegionChanged)
+    if (properties.changedProperties & RemoteLayerTreeTransaction::EventRegionChanged) {
         updateLayersForInteractionRegions(layer, properties);
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
```

Comment:
and should the whole block be guarded by `#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION) || ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)` ? Otherwise it could be dead code as it would have no side effects, right?